docs/source/extensions/loops.rst

```diff
@@ -222,7 +221,7 @@ Loop API
 --------
 Here is the full API of methods available in the Loop base class.
 
-The :class:`~pytorch_lightning.loops.base.Loop` class is the base for all loops in Lighting just like the :class:`~pytorch_lightning.core.lightning.LightningModule` is the base for all models.
```

Comment:
Might be worth grepping "Lighting" and fixing all typos at once 😂

Reply:
haha... done!!